scripts/infra/cloud-instance.sh

```diff
@@ -44,11 +44,11 @@ ec2__launch() {
     done
 
     if [ -z "$instance_id" ]; then
-        echo "Failed to launch instance"
+        echo "Failed to launch instance $INSTANCE_NAME"
```

Comment:
Not sure this is helpful to give the name the instance would have been if it had launched